addons/binding/org.openhab.binding.heos/README.md

```diff
@@ -0,0 +1,404 @@
+| Type              | String        | The type of the played media. Station or song for example             |
+| Station           | String        | The station name if it is a station (Spotify shows track name....)    |
+| Inputs            | String        | The input to be switched to. Input values from HEOS protocol          |
+| PlayUrl           | String        | Plays a media file located at the URL                                 |
+| Shuffle           | Switch        | Switches shuffle ON or OFF                                            |
+| RepeatMode        | String        | Defines the repeat mode: Inputs are: "One" ; "All" or "Off"           |
+| Playlists         | String        | Plays a playlist. Playlists are identified by numbers (starting at 0!). List can be found in the HEOS App            |
+#### Inputs depending on Player type (Date 12.02.2017):
+
+| Input names   |
+|-------------- |
+| aux_in_1      |
+| aux_in_2      |
+| aux_in_3      |
+| aux_in_4      |
+| aux1          |
+| aux2          |
+| aux3          |
+| aux4          |
+| aux5          |
+| aux6          |
+| aux7          |
+| line_in_1     |
+| line_in_2     |
+| line_in_3     |
+| line_in_4     |
+| coax_in_1     |
+| coax_in_2     |
+| optical_in_1  |
+| optical_in_2  |
+| hdmi_in_1     |
+| hdmi_arc_1    |
+| cable_sat     |
+| dvd           |
+| bluray        |
+| game          |
+| mediaplayer   |
+| cd            |
+| tuner         |
+| hdradio       |
+| tvaudio       |
+| phono         |
+
+An actual list can be found within the HEOS CLI protocol which can be found [here](http://rn.dmglobal.com/euheos/HEOS_CLI_ProtocolSpecification.pdf).
+
+### The Bridge provides the following channels:
+
+| Channel ID            | Item Type     | Description                                                                                                                                               |
+|---------------------- |-----------    |--------------------------------------------------------------------------------------------------------------------------------------------------------   |
+| Reboot                | Switch        | Reboot the whole HEOS System. Can be used if you get in trouble with the system                                                                           |
+| DynamicGroupHandling  | Switch        | If this option id activated the system automatically removes groups if they are ungrouped. Only works if the group is added via an UI.                    |
+| BuildGroup            | Switch        | Is used to define a group. The player which shall be grouped has to be selected first. If Switch is then activated the group is build.                    |
+| RawCommand            | String        | A channel where every HEOS CLI command can be send to.                                                                                                    |
```

Comment:
This isn't really nice as a channel. Would probably be nice to add it as a rule action for the Thing instead - this is something that is just about to become available with https://github.com/eclipse/smarthome/pull/6398.